README.md

```diff
@@ -0,0 +1,181 @@
+# MonkeyChat-Web-React
+
+MonkeyChat help you set up your own chat in your site.
+
+***
+
+* If you need to add your chat the fast way to do it:
+
+## MonkeyChat CDN
+Use monkeyChat.js from CDN, this MonkeyChat has MonkeyUI and MonkeySDK
+```
+https://cdn.criptext.com/v2.0.0/monkeyChat.js
+```
+#### monkeychat.init(String, String, String, String, boolean, Obj, Obj)
+To start setup chat.
+```
+monkeychat.init(IDDIV, MONKEY_APP_ID, MONKEY_APP_KEY, CONVERSATION_ID, MONKEY_DEBUG_MODE, VIEW, STYLES);
+```
+IDDIV: Id of 'div' tag where you will add the chat in your html.
+
+MONKEY_APP_ID: Monkey ID from your app.
```

Comment:
Monkey ID ! = App ID
App ID of your App, which you can get in [this admin panel](https://admin.criptext.com)